apps/nextjs/src/app/[locale]/manage/integrations/new/_integration-new-form.tsx

```diff
@@ -55,12 +55,18 @@ const formSchema = integrationCreateSchema.omit({ kind: true }).and(
 export const NewIntegrationForm = ({ searchParams }: NewIntegrationFormProps) => {
   const t = useI18n();
   const secretKinds = getAllSecretKindOptions(searchParams.kind);
+  const hasUrlSecret = secretKinds.some((kinds) => kinds.includes("url"));
   const router = useRouter();
   const [opened, setOpened] = useState(false);
+
+  let url = searchParams.url ?? getIntegrationDefaultUrl(searchParams.kind) ?? "";
+  if (hasUrlSecret) {
+    url = "http://localhost";
+  }
```

Comment:
Makes total sense how you did it 👍🏼 
I just think we should add some comments to why we for example set it to `http://localhost` when it has a url secret (which of course is because we still need to pass validation)

Thank you for the work. I like it 😄